pallets/fragments/src/lib.rs

```diff
@@ -1330,6 +1334,61 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Request to detach a **Fragment** from **Clamor**.
+		///
+		/// Note: The Fragment may actually get detached after one or more Clamor blocks since when this extrinsic is called.
+		///
+		/// Note: **Once the Fragment is detached**, an **event is emitted that includes a signature**.
+		/// This signature can then be used to attach the Proto-Fragment to an External Blockchain `target_chain`.
+		///
+		/// # Arguments
+		///
+		/// * `origin` - The origin of the extrinsic function
+		/// * `definition_hash` - **ID of the Fragment Definition** to **detach**
+		/// * `target_chain` - **External Blockchain** to attach the Proto-Fragment into
+		/// * `target_account` - **Public Account Address in the External Blockchain `target_chain`**
+		///   to assign ownership of the Proto-Fragment to
+		#[pallet::weight(25_000)] // TODO - weight
+		pub fn detach(
+			origin: OriginFor<T>,
+			definition_hash: Hash128,
+			edition_id: InstanceUnit,
+			copy_id: InstanceUnit,
+			target_chain: SupportedChains,
+			target_account: Vec<u8>, // an eth address or so
+		) -> DispatchResult {
+
+			let who = ensure_signed(origin)?;
+
+			let current_block_number = <frame_system::Pallet<T>>::block_number();
+
+			// owner must own instance
+			let owned_instances = <Inventory<T>>::get(who, definition_hash).ok_or(Error::<T>::NoPermission)?;
+			ensure!(
+				owned_instances.contains(&(Compact(edition_id), Compact(copy_id))),
+				Error::<T>::NoPermission
+			);
+
+			// REVIEW - no go if will expire this block
+			let instance = <Fragments<T>>::get((definition_hash, edition_id, copy_id))
+				.ok_or(Error::<T>::NotFound)?;
+			if let Some(expiring_at) = instance.expiring_at {
+				ensure!(current_block_number < expiring_at, Error::<T>::NotFound);
+			}
+
+			let detach_hash = DetachHash::Instance(definition_hash, Compact(edition_id), Compact(copy_id));
+			let detach_request = DetachRequest { hash: detach_hash.clone(), target_chain, target_account};
+
+			ensure!(!<DetachedHashes<T>>::contains_key(&detach_hash),Error::<T>::Detached);
+			ensure!(!<DetachRequests<T>>::get().contains(&detach_request), Error::<T>::DetachRequestAlreadyExists);
```

Comment:
@sinkingsugar is there any need to check to check if the detach requests already exists. I don't think there is, but I just added this line to be overly cautious...because why not 😅